corehq/form_processor/models.py

```diff
@@ -423,7 +423,14 @@ def read_content(self, stream=False):
         try:
             blob = db.get(self.blob_id, self.blobdb_bucket())
         except (KeyError, NotFound, BadName):
-            raise AttachmentNotFound(self.name)
+            if not self.blob_bucket and '-' in str(self.attachment_id):
+                try:
+                    # http://manage.dimagi.com/default.asp?239638
+                    blob = db.get(self.blob_id, self.blobdb_bucket(True))
```

Comment:
super nit-picky but for booleans i find it's nice to always call it as a kwarg so it's clear what the value is doing